.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: Test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
In my branch I'm using pattern match, which is a 3.10 exclusive: https://docs.python.org/3/reference/compound_stmts.html#match

Do you think we could keep it only for 3.10+ ?

Reply:
Fine for me :)